src/components/Attachments/Attachment.jsx

```diff
@@ -140,35 +141,43 @@ const Attachment = ({
           </>
         ) : (
           <>
-            <FileIcon fileName={attachment.filename} />
-            <Tooltip content={attachment.filename} position="top">
-              <Typography style="body2">{attachment.filename}</Typography>
-            </Tooltip>
+            <div
+              className="ne-attachments__attachment__preview-wrapper"
```

Comment:
```suggestion
              className="ne-attachments__preview-wrapper"
```

This class name should be good enough. Make similar changes to the sub classes.